src/components/AppFooter/ServerTime.tsx

```diff
@@ -16,7 +16,7 @@ const ServerTime = () => {
             disableHover
             tooltipContent={localFormat}
         >
-            <Text size='xs'>{UTCFormat}</Text>
+            <Text className='text-[12px]'>{UTCFormat}</Text>
```

Comment:
why not use `size='xs'`?

Reply:
Default sizes in Text component are responsive so if I pass xs as a size it will change in the mobile view which is different from the design that's why I changed it like that.

Reply:
yeah.. as of now in p2p, for these scenarios we have used isMobile ? 'sm' : 'xs' instead of hardcoding it.

Reply:
Ok then Thanks for pointing this I'll change it base on the code style 🙏